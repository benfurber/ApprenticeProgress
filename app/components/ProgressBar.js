import React, { Component } from "react";
import { Text, View } from "native-base";

class ProgressBar extends Component {
  render() {
    return (
      <View>
        <View style={{ flex: 1 }}>
          <Text>.</Text>
        </View>
      </View>
    );
  }
}

export { ProgressBar };
