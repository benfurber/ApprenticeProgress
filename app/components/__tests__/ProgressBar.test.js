// @flow

import React from "react";
import { shallow } from "enzyme";

import { ProgressBar } from "../ProgressBar";

describe("ProgressBar", () => {
  it("has a valid snapshot", () => {
    const screen = shallow(<ProgressBar score={2} />);

    expect(screen).toMatchSnapshot();
  });
});
