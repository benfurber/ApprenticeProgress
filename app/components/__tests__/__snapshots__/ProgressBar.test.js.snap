// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBar has a valid snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ProgressBar />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <Styled(ViewNB)
        style={
          Object {
            "flex": 1,
          }
        }
      >
        <Styled(Text)>
          .
        </Styled(Text)>
      </Styled(ViewNB)>,
      "virtual": undefined,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <Styled(Text)>
          .
        </Styled(Text)>,
        "style": Object {
          "flex": 1,
        },
        "virtual": undefined,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": ".",
          "virtual": undefined,
        },
        "ref": null,
        "rendered": ".",
        "type": [Function],
      },
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <Styled(ViewNB)
          style={
            Object {
              "flex": 1,
            }
          }
        >
          <Styled(Text)>
            .
          </Styled(Text)>
        </Styled(ViewNB)>,
        "virtual": undefined,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <Styled(Text)>
            .
          </Styled(Text)>,
          "style": Object {
            "flex": 1,
          },
          "virtual": undefined,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": ".",
            "virtual": undefined,
          },
          "ref": null,
          "rendered": ".",
          "type": [Function],
        },
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
